packages/x-charts/src/ChartsLegend/ChartsLegend.tsx

```diff
@@ -10,7 +10,7 @@ import { FormattedSeries, SeriesContext } from '../context/SeriesContextProvider
 import { ChartsLegendClasses, getLegendUtilityClass } from './chartsLegendClasses';
 import { DefaultizedProps } from '../models/helpers';
 import { LegendParams } from '../models/seriesType/config';
-import { ChartsText, ChartsTextStyle, getWordsByLines } from '../internals/components/ChartsText';
+import { ChartsText, ChartsTextStyle, getWordsByLines } from '../ChartsText/ChartsText';
```

Comment:
Automatic import update 😉

Reply:
Nitpick: The same would apply to all the changes. 🙈 
And whoops, I didn't notice that `getWordsByLines` is not exported from the root folder. 🙈
Maybe it would make sense to create utils or some other file and put this helper outside of the `ChartsText` component, especially when it's already used in other components..? 🤔